fr.centralesupelec.edf.riseclipse.developer.p2_to_m2/src/main/resources/riseclipse.aggr

```diff
@@ -1,25 +1,4 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<!-- 
-*************************************************************************
-**  Copyright (c) 2021 CentraleSupélec & EDF.
-**  All rights reserved. This program and the accompanying materials
-**  are made available under the terms of the Eclipse Public License v2.0
-**  which accompanies this distribution, and is available at
-**  https://www.eclipse.org/legal/epl-v20.html
-** 
-**  This file is part of the RiseClipse tool
-**  
-**  Contributors:
-**      Computer Science Department, CentraleSupélec
-**      EDF R&D
-**  Contacts:
-**      dominique.marcadet@centralesupelec.fr
-**      aurelie.dehouck-neveu@edf.fr
-**  Web site:
-**      https://riseclipse.github.io
-*************************************************************************
--->
-
```

Comment:
@marcadetd Editing the file with the CBI/aggregator and running "Clean then Build Aggregation" made the comment above disappear. Is this an issue? Is there a way to tell the aggregator to keep these comments?

Reply:
I don't know any way to keep it. I suggest adding a Readme with the copyright and the link to the tool.